core-bundle/src/InsertTag/Resolver/LegacyInsertTag.php

```diff
@@ -591,8 +591,8 @@ public function __invoke(ResolvedInsertTag $insertTag): InsertTagResult
                         $picture = $this->container->get('contao.image.picture_factory')->create($this->container->getParameter('kernel.project_dir').'/'.$strFile, $size);
 
                         $data = [
-                            'img' => $picture->getImg($this->container->getParameter('kernel.project_dir'), $staticUrl),
-                            'sources' => $picture->getSources($this->container->getParameter('kernel.project_dir'), $staticUrl),
+                            'img' => $picture->getImg($this->container->getParameter('contao.web_dir'), $staticUrl),
+                            'sources' => $picture->getSources($this->container->getParameter('contao.web_dir'), $staticUrl),
```

Comment:
```suggestion
                            'img' => $picture->getImg($this->container->getParameter('kernel.project_dir'), $staticUrl),
                            'sources' => $picture->getSources($this->container->getParameter('kernel.project_dir'), 
```
This breaks the picture insert tag.
Reverting this change makes the `{{picture::*}}` render correctly in front-end